spec/components/admin/users/tools/reactions_component_spec.rb

```diff
@@ -30,7 +30,7 @@
 
       render_inline(described_class.new(user: user))
 
-      expect(rendered_component).to have_text(reaction.category.capitalize)
+      expect(rendered_component).to have_text("Flag")
```

Comment:
Should this reference the translation `t('views.reactions.category.vomit')`, and not the string - so there's a single place to make the change? 

It's fine as is, but if we ever (1) run the test suite outside of an English locale (randomizing the language like we randomize the time) or (2) update the copy in the translation file, these tests will need to be updated.